libs/ledger-live-common/src/families/casper/hw-getAddress.ts

```diff
@@ -20,7 +20,7 @@ const resolver: Resolver = async (transport, { path, verify }) => {
   return {
     path,
     address: r.Address.length
-      ? r.Address.toString()
+      ? r.Address.toString().toLowerCase()
```

Comment:
Could be nice to rename this one letter parameter to something more explicit like that
```suggestion
      ? pubkey.Address.toString().toLowerCase()
```

Reply:
Yes, I agree... modified. What do you think?